<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formaddfumetto</class>
 <widget class="QDialog" name="formaddfumetto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aggiungi fumetto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Aggiungi fumetto:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Codice:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Titolo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Autore:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Argomento:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Data pubb.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="isbnLine"/>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="authorLine"/>
          </item>
          <item>
           <widget class="QLineEdit" name="subLine"/>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="coverLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="xml.qrc">:/img/No-Cover.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineIMGpath"/>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Descrizione:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="descText"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QPushButton" name="addNewButton">
            <property name="text">
             <string>AGGIUNGI</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="addCancelButton">
            <property name="text">
             <string>ANNULLA</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboType">
        <item>
         <property name="text">
          <string>manga</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>comicbook</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="loadFile">
        <property name="text">
         <string>scegli copertina...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="xml.qrc"/>
 </resources>
 <connections/>
</ui>
